dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetField.java

```diff
@@ -472,6 +475,13 @@ private String getUnsanitizedDisplayValue() {
     }
 
     private String getFieldDisplayValue(String value, DatasetFieldType fieldType) {
+        
+        if (fieldType.getFieldType().equals(FieldType.PERIODO)) {
+            return  "<a href=\"" + this.fieldValue + "\" target=\"_blank\">" + this.fieldValue +
+                    "</a><br>" + PeriodoDictionary.getByUrl(this.fieldValue).get()
+                            .getDetails("<br>");
+        }
```

Comment:
Are we okay with not sanitizing the values? I guess that values that come from `getDetails` and `getByUrl` are quite safe since they do not come from user. But it is not the case for `fieldValue`